<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationTabWidgetClass</class>
 <widget class="QFrame" name="ConfigurationTabWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>536</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageMainConfiguration">
      <layout class="QVBoxLayout" name="mainPageLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelConfigurationTitle">
         <property name="text">
          <string>Select a configuration section</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
